src/wp-admin/includes/class-wp-upgrader.php

```diff
@@ -798,6 +798,15 @@ public function run( $options ) {
 			)
 		);
 
+		/**
+		 * Filter result of install_package().
+		 *
+		 * @since 5.7.0
+		 * @param bool|WP_Error $result  Result from install_package().
+		 * @param array         $options Array of data for plugin/theme being updated.
+		 */
+		$result = apply_filters( 'upgrader_install_package_result', $result, $options );
```

Comment:
Seem like this should be `hook_extra` for consistency.

Reply:
I do use the `$options['destination']` in the feature plugin, in addition to `$options['hook_extra']`

Reply:
I think this needs more consideration from someone like @dd32 then. All the other hooks in `WP_Upgrader` just use `hook_extra`.

Reply:
I'm sure we could change it to `$hook_extra` and I'll calculate the destination.